capstone/src/main/webapp/locations_script.js

```diff
@@ -12,94 +12,35 @@
- 
-  fetch(link).then(response => response.text()).then((output) => {
-    let places_dict = JSON.parse(output);
-    if (places_dict['error'] === undefined) { 
-      console.log(places_dict)
-      console.log(places_dict.results)
-      let places_result = places_dict.results;
-      if (places_result.length > 0) {
-        for (let i = 0; i < places_result.length; i++) {
-          if(places_result[i].business_status !== undefined){//business, not locality
-            let id = places_result[i].name + ';' + places_result[i].geometry.location.lat + ',' + places_result[i].geometry.location.lng;
-            let marker = new google.maps.Marker({
-              position: {lat: places_result[i].geometry.location.lat, lng: places_result[i].geometry.location.lng},
-              map: map,
-              icon: 'http://maps.google.com/mapfiles/ms/icons/yellow-dot.png',
-              title: places_result[i].name,
-              id: id
-            });
-            marker_dict[id] = {'place_icon': places_result[i].icon, 'place_name': places_result[i].name,
-              'place_address': places_result[i].vicinity, 'place_lat': places_result[i].geometry.location.lat, 
-              'place_lng': places_result[i].geometry.location.lng
-            };
-            google.maps.event.addListener(marker, 'click', (function(marker, i) {
-              return function() {
-                let place = marker_dict[marker.id];
-                console.log(place)
-                console.log(place["place_address"])
-                let infowindowContent = document.getElementById('infowindow-content');
-                infowindowContent.children['place-icon'].src = place["place_icon"];
-                infowindowContent.children['place-name'].textContent = place["place_name"];
-                infowindowContent.children['place-address'].textContent = place["place_address"];
-                document.getElementById('form-place-name').textContent = place["place_name"];
-                document.getElementById('form-place-address').textContent = place["place_address"];
-                document.getElementById('form-lat').textContent = place["place_lat"];
-                document.getElementById('form-long').textContent = place["place_lng"];
-                infoWindow.setContent(infowindowContent);
-                infoWindow.open(map, marker);
-              }
-            }
-            )(marker, i));
-            establishment_markers.push(marker);
-          }
-        }
-      } else {//Google Places couldn't find places near the user, 
-        console.log('no places found')
-      }      
-    } else {//issue with the call itself
-      console.log('call failed')
-    }
-  });
-}*/
-
-/* Retrieves counties based on a passed longtiude and latitude */
+/* Retrieves counties based on a passed longtiude and latitude. */
 function getCounties(){
   console.log('getCounties');
   //getPlaces(-33.00, 151.00);
   map.setCenter({lat: -33.00, lng: 151.00}); 
```

Comment:
It's fine to do tests, but before committing, make sure these are variables. And then if you want to test locally, you can set them there.